_posts/2023-08-20-my-lfx-mentorship-experience-with-the-cncf-strimzi-community.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Navigating the Open Source Seas: My LFX Mentorship Experience with the CNCF Strimzi Community"
+date: 2023-08-20
+author: antonio_pedro
+---
+
+In the world of open-source software, collaboration, and mentorship play a pivotal role in driving innovation and fostering growth. 
+As an aspiring developer looking to dive into the intricate world of cloud-native technologies, I embarked on a remarkable journey by participating in the [LFX Mentorship](https://lfx.linuxfoundation.org/tools/mentorship) program with the [Strimzi](https://strimzi.io) community.
+This program is happens thrice a year and is organized by [The Linux Foundation](https://www.linuxfoundation.org/).
+This blog post aims to shed light on my enriching experience, highlighting the valuable insights gained, the impactful work undertaken, and the immeasurable support from the mentors and the Strimzi community.
+
+<!--more-->
+
+# LFX Mentorship and the Cloud Native Computing Foundation Strimzi Community
+
+The Linux Foundation Mentorship is a 12-week online program to empower students to contribute to open-source projects under the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
+On the other hand, Strimzi is a [Cloud Native Computing Foundation](https://www.cncf.io/) sandbox project that focuses on providing operators for running Apache Kafka on Kubernetes.
+With my interest piqued by this innovative endeavor, I decided to join the Strimzi community through the LFX Mentorship program.
+
+The LFX mentorship program bridges budding developers and experienced mentors, providing an avenue to contribute meaningfully to open-source projects.
+
+# From where everything started
+
+Everything started in 2022, I was looking for things I could do during my summer holidays.
+I investigated some mentorship programs such as Google Summer of Code, MLH Fellowship, Outreachy, and LFX Mentorship.
+
+## The Application process
+
+I have selected two projects for which my skill set fits the recommended skills.
+Among them, the Strimzi project was there waiting for me. 
+
+`“CNCF - Strimzi: Proof of Concept of an MQTT to Apache Kafka bridge for producing messages.”`
+
+This project got my attention among all the projects because I had practical experience with MQTT from my undergraduate program.
```

Comment:
I would add the link to the project on the LFX website: https://mentorship.lfx.linuxfoundation.org/project/8d301adf-94d8-4e5d-821d-f904ed15c3f9

Reply:
I agree.